docs/concepts/dotnet-standard-library.md

```diff
@@ -0,0 +1,130 @@
+.NET Standard Library
+=====================
+
+The .NET Standard Library is a formal specification of .NET APIs that are intended to be available on all .NET runtimes. The motivation behind the Standard Library is establishing greater uniformity in the .NET ecosystem. [ECMA 335](https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/dotnet-standards.md) continues to establish uniformity for .NET runtime behavior, but there is no similar spec for the .NET Base Class Libraries (BCL) for .NET library implementations. 
```

Comment:
Agreed. I knew that when I wrote it. I chose to ignore that fact for exactly the reason you state. It also seems like unnecessary complexity to say "well, we spec'd the APIs but not in a meaningful way, so it wasn't really a useful spec. the new one will be the spec."